src/vs/platform/environment/node/environmentService.ts

```diff
@@ -114,6 +114,9 @@ export class EnvironmentService implements IEnvironmentService {
 	@memoize
 	get sharedIPCHandle(): string { return `${getIPCHandlePrefix()}-${pkg.version}-shared${getIPCHandleSuffix()}`; }
 
+	@memoize
+	get nodeCachedDataDir(): string { return path.join(this.userDataPath, 'CachedData', pkg.version); }
```

Comment:
@jrieken should we not use the commit ID of the built as a way to identify the backup? if we use the version, wouldn't we use old cached data each time we update insiders build?